lib/spack/docs/getting_started.rst

```diff
@@ -178,7 +178,7 @@ Spack fall back to bootstrapping from sources:
 .. code-block:: console
 
    $ spack bootstrap disable github-actions-v0.3
-   ==> "github-actions-v0.2" is now disabled and will not be used for bootstrapping
+   ==> "github-actions-v0.3" is now disabled and will not be used for bootstrapping
```

Comment:
Is this true? It looks enabled and still listed above.

Reply:
I also have to update it when this is done to add the new repo.

Reply:
Gotcha.